cookbooks/travis_build_environment/recipes/sysctl.rb

```diff
@@ -29,3 +29,20 @@
   only_if { node['travis_build_environment']['sysctl_disable_ipv6'] }
   notifies :run, 'execute[update sysctl travis-disable-ipv6]'
 end
+
+execute 'update sysctl travis-enable-ipv4-forwarding' do
+  command 'sysctl -p /etc/sysctl.d/99-travis-enable-ipv4-forwarding.conf'
+  action :nothing
+end
+
+file '/etc/sysctl.d/99-travis-enable-ipv4-forwarding.conf' do
+  content "net.ipv4.ip_forward = 1"
+  content <<-IPV4_CONFIG.gsub(/^\s+> ?/, '')
```

Comment:
You should be able to safely replace this `<<-MARKER.gsub...` with `<<~MARKER` since we're running chef on ruby 2.3+ everywhere now (and if we're not, I would like to fix it).

Reply:
Is the double `content` declaration because of the need for a newline?

Reply:
Nope, that's a mistake! But I'd be curious to know if `content "net.ipv4.ip_forward = 1"` by itself would work!

Reply:
The string form should work fine, imho! I would add a newline literal for paranoia fun, so:

``` ruby
# ...
    content "net.ipv4.ip_forward = 1\n"
```